R/vendor.R

```diff
@@ -150,6 +150,9 @@ renv_vendor_imports <- function() {
 renv_vendor_sources <- function(version = renv_metadata_version()) {
 
   tarball <- renv_bootstrap_download_github(version)
```

Comment:
Maybe we should alter this function so that it either returns a path, or emits an error (instead of FALSE)?

Reply:
Any option is better than attempting to `untar(FALSE)` 🤣

Do you want to make those changes separately or do you want me to iterate, here?

Reply:
I think that's hard to do without changing how `renv_bootstrap_download()` works (along with all other `renv_bootstrap_download_*` functions). It's a bit icky, but I think given that the vendor code is used by only a few people, we should just go with the code in this PR.